third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -806,14 +780,39 @@ func TestAccContainerCluster_withNodeConfigScopeAlias(t *testing.T) {
 			},
 			{
 				ResourceName:		 "google_container_cluster.with_node_config_scope_alias",
-				ImportStateIdPrefix: "us-central1-f/",
 				ImportState:		 true,
 				ImportStateVerify:	 true,
 			},
 		},
 	})
 }
 
+<% unless version.nil? || version == 'ga' -%>
```

Comment:
This got deleted in https://github.com/GoogleCloudPlatform/magic-modules/pull/2523